sdcm/cluster.py

```diff
@@ -121,6 +122,12 @@ class Setup:
     BACKTRACE_DECODING = False
     INTRA_NODE_COMM_PUBLIC = False
     RSYSLOG_ADDRESS = None
+    LDAP_ADDRESS = None
```

Comment:
why do need all those on the Setup class ? they are going to change during the test ?

Reply:
they will be populated once the container is created, and will be used across the test in different places...
the `LDAP_ADDRESS` will be used for create a connection to the LDAP server for example...

Reply:
the `LDAP_ADDRESS` o.k. but the rest are hardcodede values for now, like username/password and such. I don't see the reason for them to be on Setup class.

in some places you are using `LDAP_BASE_OBJECT` directly, and in some reading it from ` node.ldap_config['ldap_base_object']` and in some from `Setup.LDAP_BASE_OBJECT`

I would prefer it would be the first one only, until prove it's need to be dynamic or change during the run.
i.e. all those are known in advance, no need to put them all over the place.

Reply:
the only access to `node.ldap_config['ldap_base_object']` is when we set the `Setup.LDAP_BASE_OBJECT`, so its usage is pretty consistent

Reply:
you are missing the point, having one `LDAP_BASE_OBJECT` that all the code point to it, if it's a const value.
no need for all the wrapper around it.

`Setup` should be used for global values we are changing during he span of test, i.e. like addresses and ports.

Please remove any consts values out of `Setup` class.